### ISSUE TITLE
PNI home: update intro text

### ISSUE THREAD

**Issue by kristinashu** [mozilla/foundation.mozilla.org#4021]
- [ ] Add new copy, icon, and link to the intro text.

**New copy** 
Look for this symbol [icon] for products that meet our [Minimum Security Standards](https://foundation.mozilla.org/en/privacynotincluded/about/meets-minimum-security-standards).

**Comp**
<img width="760" alt="image" src="https://user-images.githubusercontent.com/1068783/70285662-1e01b280-177d-11ea-96e2-6787f6e13ab6.png">

**🗒  Design -> Dev Handoff Checklist**
- [x] Design has been finalized
- [x] Link to original design ticket _#3902_
- [x] Link to Redpen _https://redpen.io/p/mn3629c03f84209e3b_
- [x] Design changes been merged into the master Sketch file
- [x] Desktop and mobile mockups are included
- [x] Assets are included _(same icon is already used on the page)_
- [x] Hover state for elements are designed _(#0d10bf same functionality as the "Learn more" link on product pages)_

cc @beccaklam

**Comment by mmmavis** [comment]
<img width="933" alt="image" src="https://user-images.githubusercontent.com/2896608/70668441-7a0a8200-1c28-11ea-81be-a21e364ab4fe.png">

Hyperlink in `.body-large` by default is in blue and the underline only appears when it's on hover. Do we want to override this for all hyperlinks in `.body-large` on PNI? And what should the hover state be?

**Comment by mmmavis** [comment]
@beccaklam , 

I know PNI is a special project so it doesn't align with Foundation style guide 100%. I'd like to know if we plan to slowly make styling adjustments so that PNI has fewer style overrides over time. If that's not the plan, I'll just assume by default we want to add style override to the codebase so that things look like what has been designed in the Sketch file. This will help reduce the back-and-forth checks between devs and designers!

**Comment by beccaklam** [comment]
> <img alt="image" width="933" src="https://user-images.githubusercontent.com/2896608/70668441-7a0a8200-1c28-11ea-81be-a21e364ab4fe.png">
> 
> Hyperlink in `.body-large` by default is in blue and the underline only appears when it's on hover. Do we want to override this for all hyperlinks in `.body-large` on PNI?

Let's not override and keep the hyperlink blue, with the underline only showing on hover!

**Comment by beccaklam** [comment]
> @beccaklam ,
> 
> I know PNI is a special project so it doesn't align with Foundation style guide 100%. I'd like to know if we plan to slowly make styling adjustments so that PNI has fewer style overrides over time. If that's not the plan, I'll just assume by default we want to add style override to the PNI codebase so that things look like what has been designed in the Sketch file. This will help reduce the back-and-forth checks between devs and designers!

I think PNI was developed before we started overhauling the brand guidelines so I think it's okay for you not to override the styles. I'll try to adjust the design during QA if anything looks too wonky. In the future for special projects could we have a separate style sheet for special cases?

**Comment by mmmavis** [comment]
> I think PNI was developed before we started overhauling the brand guidelines so I think it's okay for you not to override the styles. I'll try to adjust the design during QA if anything looks too wonky.

Sounds good!

> In the future for special projects could we have a separate style sheet for special cases?

Technical-wise yes! I just wasn't sure what our plan was for PNI styling alignment that's why I raised the overrirde issue here.
